src/sub_8002799C.c

```diff
@@ -0,0 +1,34 @@
+
+#include "psyq.h"
+#include "libdg/inline_n.h"
+
+void sub_8002799C(int a0)
+{
+    int v1;
+    int v0;
+
+    if (a0 == 1)
+    {
+        v0 = *(short *)0x1F80004C;
+        v1 = *(short *)0x1F800034;
+        v0 -= v1;
+        v1 = *(short *)0x1F800048;
+    }
+    else
+    {
+        v0 = *(short *)0x1F80004E; // TODO: is this a part of (SVECTOR *)0x1f800040 ???, same case above/below
```

Comment:
@ChrizNuh On second though (it was me who originally wrote this comment), this address is too far from 0x1f800040. I'll merge this PR and delete this comment when I push my changes. When all functions using scratchpad are matched we should probably go through all those addresses and name them nicely.